src/KeyboardManager.vala

```diff
@@ -17,43 +17,38 @@
 
 namespace Gala {
     public class KeyboardManager : Object {
-        static KeyboardManager? instance;
-        static VariantType sources_variant_type;
+        private static VariantType sources_variant_type;
 
-        public static void init (Meta.Display display) {
-            if (instance != null)
-                return;
-
-            instance = new KeyboardManager ();
-
-            display.modifiers_accelerator_activated.connect (instance.handle_modifiers_accelerator_activated);
-        }
+        public weak Meta.Display display { get; construct; }
+        private GLib.Settings? settings;
```

Comment:
This may not get initialised explicitly - is it OK to assume it gets automatically initialised to `null`?